docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.0-M8.md

```diff
@@ -0,0 +1,771 @@
+TCK Results
+===========
+
+
+As required by the
+[Eclipse Foundation Technology Compatibility Kit License](https://www.eclipse.org/legal/tck.php),
+following is a summary of the TCK results for releases of Jakarta EE Platform 10.0, certification summary.
+
+# Jakarta EE Platform 10.0, Eclipse GlassFish 7.0.0-M8, TCK Certification Summary
+
+- Product Name, Version and download URL (if applicable): <br/>
+  Eclipse GlassFish 7.0.0-M8: [glassfish-7.0.0-M8.zip](https://download.eclipse.org/ee4j/glassfish/glassfish-7.0.0-M8.zip)
+
+- Specification Name, Version and download URL: <br/>
+  [Jakarta EE Platform, 10.0](https://jakarta.ee/specifications/platform/10/)
+
+- TCK Version, digital SHA-256 fingerprint and download URL: <br/>
+  [Jakarta EE Platform TCK 10.0](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip), 
```

Comment:
This does not seem to be reachable at such location. (I suppose it's the target address for final zip.)
But it matches https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/promoted/eftl/jakarta-jakartaeetck-10.0.0-tckinfo.txt:
```
***********************************************************************************
***                        TCK bundle information                               ***
*** Name:       jakarta-jakartaeetck-10.0.0.zip                                     ***
*** Bundle Copied to URL:    https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/promoted/eftl//jakarta-jakartaeetck-10.0.0.zip ***
*** Date and size: date: 2022-08-16 16:37:36.000000000 +0000, size(b): 527767518        ***
*** SHA256SUM: 7874244f69ed9c72f4d4b5a8c48b21d39c56f4b877b1d4d735247a7f08215a5e ***
***                                                                             ***
***********************************************************************************
```

Reply:
Correct, https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip will not be available until after the EE 10 release is final.

Reply:
https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-jakartaeetck-10.0.0.zip is available now but not long term.